arch-design/dashboard/requirements_analysis_for_i_release.md

```diff
@@ -0,0 +1,124 @@
+# SODA Dashboard requirements for I release
+
+  
+
+**Author(s)**: [Anvith KS](https://github.com/anvithks)
+
+Major Version Updates
+
+|**Date** | **Version** | **Description** |
+|---------|-------------|-----------------|
+| 25th February 2021  | v0.1  | This document lists the requirements and changes needed for SODA Dashboard for I release. |
+
+
+
+## Goals
+- Multicloud UI 
+    - Support Heterogenous object storage
+    - Bucket Create / Update / Delete on cloud storage
+    - Bucket tiering based on policy
+    - Bug fixes and improvements
+- Delfin UI updates
+    - Add new storage driver registration support 
+    - Add new methods of access to Drivers
+        - CLI
+        - SMI-S
+    - Change the configure performance metrics API as per the changes made to Delfin.
+    - Add new resource types for only NAS storage device  to support NAS capabilities
+        - Qtree
+        - File Systems
+        - File share   
+- Bug fixes and improvements
+- SODA Dashboard support for HA
+
+
+## Motivation and background
+
+The handling of buckets on cloud storage using SODA Multicloud APIs is undergoing a design change as per this pr [architecture-analysis/99](https://github.com/sodafoundation/architecture-analysis/pull/99). This will change the fundamental design of how SODA Multicloud deals with registration of backends and buckets. This requires some modifications in API parameters and some changes to how the Dashboard lists and operates on backends, buckets and objects within those buckets. SODA Delfin is adding new drivers support on a regular basis and this requires the Dashboard to be updated to allow the user to register storages for these new drivers.Additionally any bug fixes and improvements will be covered as part of this release
+
+
+### Requirement Analysis
+
+
+#### Multicloud - Bucket Management
+---
+* Create bucket on cloud backend (AWS, Azure, GCP and all supported vendors) based on custom tier policy
```

Comment:
These are mainly the functional requirements. Requirement should be high level

Reply:
I think the high level requirements are listed in the Goals section. This is the Analysis section so there are some more details.